waku/common/rate_limiter.go

```diff
@@ -25,10 +25,11 @@ import (
 	"net"
 	"time"
 
-	"github.com/tsenart/tb"
+	"golang.org/x/crypto/sha3"
 
 	"github.com/ethereum/go-ethereum/p2p"
 	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/tsenart/tb"
```

Comment:
Ah, I think that the linter is complaining that your imports aren't grouped by repo.

```suggestion
	"golang.org/x/crypto/sha3"

	"github.com/ethereum/go-ethereum/p2p"
	"github.com/ethereum/go-ethereum/p2p/enode"

	"github.com/tsenart/tb"
```

Reply:
The linter gives the worst messages for this kind of linting issue.

Reply:
I think what the linter wants is:

```
- Imports from stdlib

- Imports from 3rd party vendors

- Imports from your own project
```